uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/instances/calendar-instance.js

```diff
@@ -33,6 +31,29 @@ define([
       return ModelInstance.extend({
 
         /**
+         * Creates a new instance, optionally with its data normalized.
+         * Also, it will allow children to add/remove/update current attributes of
+         * the instance using transformAttributes method
+         *
+         * @param {object} data - The instance data
+         * @return {object}
+         */
+        init: function (data) {
+          var iterator,
```

Comment:
Please even if in the end you do need it (as in, you can't use `forEach`, or `map, or similar), call it `i`